dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+@Version(Constants.OSGI_VERSION)
+@TraceOptions(traceGroup = Constants.TRACE_GROUP, messageBundle = Constants.TRACE_OPENAPI)
+package io.openliberty.microprofile.openapi20;
```

Comment:
This isn't how you're supposed to version packages.

Should be `1.0` in the release, with the major and minor versions being updated for future releases in accordance with semantic versioning.

Reply:
So... simply change `Constants::OSGI_VERSION` to be "1.0"?

Reply:
For now that will do, going forwards, packages are versioned individually, so they're unlikely to have the same version.